extensions/modrinth-search/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "modrinth-search",
+  "title": "Modrinth Search",
+  "description": "Searches Modrinth.com for anything listed.",
+  "icon": "command-icon.png",
+  "author": "legenden",
+  "license": "MIT",
+  "access": "public",
+  "categories": [
+    "Media",
+    "Web"
+  ],
+  "commands": [
+    {
+      "name": "search-mods",
+      "title": "Search Mods",
+      "description": "Searches mods on Modrinth.com",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.69.0",
+    "got": "^11.8.2"
```

Comment:
Got 11 is a bit outdated. I understand that you choose because of the CommonJS. But you can consider updating the project to ESM.

You can refer to my project configuration to see how to use ESM:

- tsconfig.json: https://github.com/raycast/extensions/blob/main/extensions/simple-icons/tsconfig.json
- package.json: https://github.com/raycast/extensions/blob/main/extensions/simple-icons/package.json

Reply:
I've opted for the `useFetch` call inside the utils package instead, since it came with fetch and cache already built in, and functions a lot like react-query :D

Reply:
Then the Got can be removed.

Reply:
That is my bad, I thought I ran npm uninstall, but apparently not.

It has now been removed, thank you :D